CHANGELOG.md

```diff
@@ -56,6 +56,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
     custom security group mappings
 - **CUMULUS-2402**
   - Officially expose `sftp()` for use in `@cumulus/sftp-client`
+- **CUMULUS-2355**
```

Comment:
This CHANGELOG note is nested under the 6.0.0 release right now, but it should be moved up to the "Unreleased" section